docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -310,24 +310,26 @@ that match the index pattern.
 when the rollover action is triggered for an index.
 
 You can use the {kib} Create template wizard to add the template. To access the
-wizard, open the menu, go to *Stack Management > Index Management*, and click
-the *Index Templates* tab.
+wizard, open the menu and go to *Stack Management > Index Management*. In the
+the *Index Templates* tab, click *Create template*.
 
 [role="screenshot"]
-image:images/ilm/create-template-wizard.png[]
+image:images/ilm/create-template-wizard.png[Create template page]
 
 The create template request for the example template looks like this:
 
 [source,console]
 -----------------------
-PUT _template/timeseries_template
+PUT _index_template/timeseries_template
 {
   "index_patterns": ["timeseries-*"],                 <1>
-  "settings": {
-    "number_of_shards": 1,
-    "number_of_replicas": 1,
-    "index.lifecycle.name": "timeseries_policy",      <2>
-    "index.lifecycle.rollover_alias": "timeseries"    <3>
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 1,
+      "index.lifecycle.name": "timeseries_policy",      <2>
+      "index.lifecycle.rollover_alias": "timeseries"    <3>
```

Comment:
If we update these docs to use data streams at the same time, we could add that here to this template and then remove this line, what do you think?

Reply:
There are instructions for using data streams with ILM further up the page. Those sections already use composable templates.

This section is nested under "Manage time-series data without data streams":
https://www.elastic.co/guide/en/elasticsearch/reference/master/getting-started-index-lifecycle-management.html#manage-time-series-data-without-data-streams

We decided to keep these instructions around because:

1. From what I understand, Logstash will continue using rollover aliases.
2. Data streams aren't a good fit for users that need to frequently add or update docs.

There's a tip at the top of the page that directs users here if #2 is the case:
https://www.elastic.co/guide/en/elasticsearch/reference/master/getting-started-index-lifecycle-management.html#getting-started-index-lifecycle-management

If neither of those are applicable anymore, I think we can just remove rollover alias instructions entirely.

Reply:
Sorry, I totally missed that part, LGTM then!